ehrql/serializer.py

```diff
@@ -39,6 +39,7 @@ class SerializerError(Exception):
         DisclosureControlConfig,
         *get_all_subclasses(Node),
         *get_all_subclasses(BaseCode),
+        *get_all_subclasses(BaseMultiCodeString),
```

Comment:
This is all you need to do serializer-wise, because we never need to serialize _instances_ of this type. We just need to serialize references to the type itself because they occur in the schema.